app/controllers/KairosDbProxyController.java

```diff
@@ -173,6 +187,29 @@ public KairosDbProxyController(
         }
     }
 
+    MetricsQuery addMergeAggregator(final MetricsQuery metricsQuery) {
+        final List<Metric> newMetrics = new ArrayList<>();
+        for (Metric metric : metricsQuery.getMetrics()) {
+            List<Aggregator> newAggregators = new ArrayList<>();
+            if (!metric.getAggregators().isEmpty() && !(metric.getAggregators().get(0).getName()).equals("merge")
```

Comment:
Should this condition be moved into a function? Probably make it easier to test as well.